TappyReader/SerialTappy.swift

```diff
@@ -61,7 +62,38 @@ public class SerialTappy : NSObject, Tappy {
                 if(decodedPacket.count != 0){
                     do{
                         let message : RawTCMPMesssage = try RawTCMPMesssage(message: decodedPacket)
-                        responseListener(message)
+                        // need to resolve the message first
```

Comment:
This can't be incorporated here in `SerialTappy` since at this level, the '`SerialTappy` has no knowledge of explicit command families, response codes, etc..

Reply:
I'll remove this from here in my next pull request. When I pass the payload back as JSON I am able to get the ndef text on the nativescript side